template/.github/workflows/deploy_preview.yml

```diff
@@ -0,0 +1,26 @@
+name: Deploy Preview to Netlify
+
+on: [pull_request, workflow_dispatch]
+
+jobs:
+  deploy-preview:
+    runs-on: ubuntu-18.04
```

Comment:
I think we should add the name attribute for this job as well

Reply:
50c2b66 ✅